### PR TITLE
Adapt to change in onet for no base64 in toml files

### PR DESCRIPTION
**Pull request by jeffallen**
Automatically convert public.toml files using the migrate
feature of run_conode.sh.

### CODE REVIEW

File: conode/conode.go [hunk 1]
```diff
@@ -128,3 +133,29 @@ func checkConfig(c *cli.Context) error {
 	}
 	return check.Config(tomlFileName, c.Bool("detail"))
 }
+
+// Convert toml files from base64-encoded kyes to hex-encoded keys
+func convert64(c *cli.Context) error {
+	scanner := bufio.NewScanner(os.Stdin)
```
Comment (ineiti) [inline conode/conode.go]: Please toml-read the file as a `app.GroupToml` and work on that...

And there should be a `private.toml` somewhere - shouldn't that be also converted?

File: conode/run_conode.sh [hunk 2]
```diff
@@ -118,16 +122,22 @@ runLocal(){
...
+			mv $co/public.toml $co/public.toml.bak
+			$CONODE_BIN convert64 < $co/public.toml.bak > $co/public.toml
+		    fi
 		fi
 
 		if [ ! -d $co ]; then
 			echo -e "127.0.0.1:$((7000 + 2 * $n))\nConode_$n\n$co" | $CONODE_BIN setup
 		fi
-		$CONODE_BIN -c $co/private.toml -d $DEBUG &
+		$CONODE_BIN server -c $co/private.toml -d $DEBUG &
```
Comment (ineiti) [inline conode/run_conode.sh]: shouldn't the `private.toml` also be migrated?

File: conode/run_conode.sh [hunk 3]
```diff
@@ -255,6 +267,14 @@ migrate(){
 			fi
 			echo $VERSION > $PATH_VERSION
 			;;
+		1.2-1)
```
Comment (ineiti) [inline conode/run_conode.sh]: I really think this should not be `v1`, but rather `v2-alpha` or so. We're breaking existing code here!

File: conode/conode.go (hunk 1 above)
Comment (jeffallen) [reply conode/conode.go]: Private was always in hex. Thus part of the point of this fix to reduce the confusion.

**Pull request merged**